flycheck.el

```diff
@@ -7518,6 +7513,14 @@ See URL `https://github.com/eslint/eslint'."
                                    (flycheck-error-message err))))
                           (flycheck-sanitize-errors errors))
                   errors)
+  :enabled (lambda (checker)
+             (let* ((directory (concat
+                                (flycheck-compute-working-directory checker)
+                                "."))
```

Comment:
Did you try to change the working directory and just pass `.` to eslint?  IOW, like this:

```
(let* ((default-directory (flycheck-compute-working-directory checker))
          …
         (exitcode (call-process executable nil nil nil "--print-config" ".")))
   …)
```

Wouldn't that work as well?

I think that's easier to understand, and since you're computing the working directory anyway why not use it as such?